pages/common/redis-server.md

```diff
@@ -3,7 +3,7 @@
 > Persistent key-value database.
 > More information: <https://redis.io>.
 
-- Start Redis server, using the default port (6379), and write logs to `stdout`:
+- Start Redis server, using the default port (`6379`), and write logs to stdout:
```

Comment:
I think something has gotten confused here. I thought we were removing the `backticks` around port numbers?

Reply:
I'm unsure of what happened but it is good that you've noticed that.